From 20d37755062eec764cbabf5bab803164be81d4df Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 15 Feb 2017 10:12:34 +0200
Subject: [PATCH 1406/2241] fix: mvebu: defconfig: increase memory area for CMA

Issue:
- In mvpp2x net driver RXQ and TXQ allocated by dma_alloc_coherent to
   stay in 32 bit memory space.
- If we use multi queue RSS mode + 8 CoS queues there isn't enough
  memory to start all 6 ports.

Fix:
- Increase memory area for contiguous memory allocator
  from default 16MB to 64MB

Change-Id: I710bd5efda081286d5a98f9767672d29e3674109
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37125
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 2dd54f4..84e8417 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -75,6 +75,7 @@ CONFIG_UEVENT_HELPER_PATH="/sbin/hotplug"
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 CONFIG_DMA_CMA=y
+CONFIG_CMA_SIZE_MBYTES=64
 CONFIG_VEXPRESS_CONFIG=y
 CONFIG_MTD=y
 CONFIG_MTD_CMDLINE_PARTS=y
-- 
2.7.4

